openhands-cli/openhands_cli/locations.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+
+LLM_SETTINGS_PATH = "~/.openhands/llm_settings.json"
```

Comment:
Should we support making this configurable and if so how? Previously we'd use toml but I'd like to avoid it going forward.